zlibWrapper/examples/zwrapbench.c

```diff
@@ -130,7 +130,7 @@ void BMK_SetBlockSize(size_t blockSize)
 **********************************************************/
 typedef struct
 {
-    const char* srcPtr;
+    char* srcPtr;
```

Comment:
made a test by adding `#define ZLIB_CONST` before `#include <zlib.h>` within `zstd_zlibwrapper.h`, fixed the `const` issue within `zwrapbench`.